examples/test/test_ai_society_example.py

```diff
@@ -14,9 +14,15 @@
 import pytest
 
 import examples.ai_society.role_playing
+import examples.function_call.role_playing_with_function
 from camel.typing import ModelType
 
 
 @pytest.mark.slow
 def test_ai_society_role_playing_example():
     examples.ai_society.role_playing.main(ModelType.STUB)
+
+
+@pytest.mark.slow
```

Comment:
Is it slow?

Reply:
emm this will cause a few rounds of chatting but I am not sure how to determine whether a test should be slow or not. Is there a threshold for this?

Reply:
Let's have 3 seconds as a cutoff. How long does the test take?

Reply:
Then it should be marked as slow as this will last for more than 1 minute